service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellClient.java

```diff
@@ -19,6 +19,8 @@ public class CromwellClient {
   private final DependencyUrlLoader dependencyUrlLoader;
 
   private final OkHttpClient singletonHttpClient;
+  private static final String CONNECTION = "Connection";
+  private static final String CLOSE = "close";
```

Comment:
Super minor, but it feels a little weird to me that we're storing these as apparently unconnected constants... I wonder whether a neater way to DRY without these looking quite so "magic" could be with a function to set the client up, eg:
```
public setClientConnectionToClose(ApiClient apiClient) {
  apiClient.addDefaultHeader("connection", "close");
}
```

... what do you think?

Reply:
Or you could even generalize it to something like `setupApiClient` and throw in the  debug api logging at the same time